guidelines/sc/22/consistent-help.html

```diff
@@ -5,7 +5,7 @@ <h4>Consistent Help</h4>
     <p class="conformance-level">A</p>
 	<p class="change">New</p>
     
-    <p>For each <a>web page</a> within a <a>set of web pages</a> that provides one or more of the following ways of finding help, access to at least one form of help is included in the same relative order on each page:</p>
+    <p>For each <a>web page</a> within a <a>set of web pages</a> that provides one or more of following ways of finding help somewhere within that set, access to at least one form of help is included in the same relative order on each page within the set of web pages if it is included on that page:</p>
```

Comment:
`one or more of following ways` sounds off. i'd keep the original `one or more of the following ways`